src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -16,29 +17,39 @@ namespace Microsoft.FeatureManagement
     /// <summary>
     /// Used to evaluate whether a feature is enabled or disabled.
     /// </summary>
-    class FeatureManager : IFeatureManager
+    class FeatureManager : IFeatureManager, IFeatureVariantManager
     {
         private readonly IFeatureDefinitionProvider _featureDefinitionProvider;
         private readonly IEnumerable<IFeatureFilterMetadata> _featureFilters;
+        private readonly IEnumerable<IFeatureVariantAssignerMetadata> _variantAssigners;
+        private readonly IFeatureVariantOptionsResolver _variantOptionsResolver;
         private readonly IEnumerable<ISessionManager> _sessionManagers;
         private readonly ILogger _logger;
         private readonly ConcurrentDictionary<string, IFeatureFilterMetadata> _filterMetadataCache;
+        private readonly ConcurrentDictionary<string, IFeatureVariantAssignerMetadata> _assignerMetadataCache;
```

Comment:
>private readonly ConcurrentDictionary<string, IFeatureVariantAssignerMetadata> _assignerMetadataCache [](http://example.com/codeflow?start=8&length=101)

For all the ConcurrentDictionaries in this file, we are using default string comparison (Ordinal). Does that mean filters/assigners are case-sensitive?

Reply:
Yes, they are case insensitive.

Reply:
I responded to that oddly because I misread the question. They are case insensitive, and it makes sense to update the dictionaries to do case insensitive comparison.

Reply:
Updated.